daniel-cabero/lesson_4/home_work/Character.java

```diff
@@ -0,0 +1,13 @@
+package lesson_4.home_work;
+
+/**
+ * @author Daniel Cabero
+ * @since 09/12/2018.
+ */
+public abstract class Character {
+    String name;
+    int heal;
+    Attack attack;
+
+    public abstract int attack();
```

Comment:
this class should implement IAttack interface, leaving that method as abstract to be implemented in child classes